fedimint-core/src/config.rs

```diff
@@ -94,11 +94,11 @@ impl JsonWithKind {
 }
 
 #[derive(Debug, Clone, Eq, PartialEq, Hash, Serialize, Deserialize, Encodable)]
-pub struct ApiEndpoint {
-    /// The peer's API websocket network address and port (e.g.
+pub struct PeerUrl {
+    /// The peer's public URL (e.g.
     /// `ws://10.42.0.10:5000`)
     pub url: Url,
```

Comment:
nit: can we move that on one line and change the websocket IP to an actual *public* one? :sweat_smile: it creates dissonance in my head reading 'public URL' but seeing an IP from 10/8